src/shared/entities/player.shared.ts

```diff
@@ -11,8 +11,8 @@ export class SharedPlayer {
 
     static protocol = {
         id: { type: nengi.ASCIIString },
-        x: {type: nengi.Int16 },
-        y: {type : nengi.Int16 },
+        x: { type: nengi.Number, interp: true },
+        y: { type: nengi.Number, interp: true },
```

Comment:
The actual interp doesn't seem to be functioning quite right, as movement is still incredibly jerky. I did not inspect the actual x/y values coming over the wire, though.